SWIGGYDEMO_REACT/swiggy-react-app/src/components/itemDetailsAndCart/components/listByCategory/components/items/components/item/components/actionButtonForItems/ActionButtonForItemsContainer.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ActionButtonForItems from "./ActionButtonForItems";
+
+function ActionButtonForItemsContainer ({pk, count, onClickPlus, onClickMinus}) {
+
+    function increaseCountItems (pk) {
+        onClickPlus(pk);
+    }
+
+    function decreaseCountItems (pk) {
```

Comment:
no operation wrapper function, please use the called function directly

Reply:
Called function directly in onClick